src/main/java/gregtech/common/metatileentities/multi/electric/MetaTileEntityFusionReactor.java

```diff
@@ -112,7 +112,7 @@ protected BlockPattern createStructurePattern() {
                 .where('K', states(getCoilState()))
                 .where('O', states(getCasingState()).or(abilities(MultiblockAbility.EXPORT_FLUIDS)))
                 .where('A', air())
-                .where('I', states(getCasingState()).or(abilities(MultiblockAbility.IMPORT_FLUIDS).setMinGlobalLimited(recipeMap.getMinFluidInputs())))
+                .where('I', states(getCasingState()).or(abilities(MultiblockAbility.IMPORT_FLUIDS).setMinGlobalLimited(1)))
```

Comment:
This could probably just be 2, since all fusion reactor recipes need 2 fluids anyway.